src/main/kotlin/algorithms_6/Algorithms.kt

```diff
@@ -1,9 +1,39 @@
 package algorithms_6
 
+import algorithms_6.algorithms.FibonacciAlgorithm
+import algorithms_6.algorithms.HanoiTowerAlgorithm
+import algorithms_6.algorithms.QuicksortAlgorithm
+import optimazation_13.utils.formatToTimeString
+import java.util.*
+import kotlin.system.exitProcess
+
 class Algorithms {
 
 }
 
 fun main() {
-    println("Algorithms")
-}
\ No newline at end of file
+    println("[1] Ciąg Fibonacciego")
+    println("[2] Wieże Hanoi")
+    println("[3] Quicksort")
+    print("Wybierz algorytm: ")
+
+    val reader = Scanner(System.`in`)
+
+    val algorithm = when (reader.nextInt()) {
+        1 -> FibonacciAlgorithm()
+        2 -> HanoiTowerAlgorithm()
+        3 -> QuicksortAlgorithm()
+        else -> exitProcess(1)
+    }
```

Comment:
When something different than a number is inputted the execution crashes. Maybe you can use a method akin to getCorrectUserInput in the Optimization class